addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -39,6 +39,11 @@
         $color: str-replace("#{o-color($mapped-color)}", '#', '%23');
         $url-params: '#{$url-params}&c#{$i}=#{$color}';
     }
+    // Replace invalid characters used for rgba colors of some theme shapes
+    // (e.g. rgba(0, 0, 0, .5)).
+    $url-params: str-replace($url-params, '(', '%28');
+    $url-params: str-replace($url-params, ')', '%29');
+    $url-params: str-replace($url-params, ',', '%2C');
```

Comment:
- A bit strange to replace the # color by color then replacing the rest afterwards. Maybe create an `encode-color` function?
- Don't we have to encode the potential spaces too then?

Reply:
> * Don't we have to encode the potential spaces too then?

It worked because modern browsers automatically adapt spaces in URLs, but I also added it to the function to be sure.